WordPress/WordPressShareExtension/PostStatusPickerViewController.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+class PostStatusPickerViewController : UITableViewController
+{
+    // MARK: - Initializers
+    init(statuses : [String: String]) {
+        assert(statuses.count > 0, "Let's show at least one status!")
+        
+        // Note:  We'll store the sorted Post Statuses, into an array, as a (Key, Description) tuple.
+        self.sortedStatuses = statuses.sort { $0.1 < $1.1 }
+        super.init(style: .Plain)
+    }
+    
+    required init?(coder: NSCoder) {
```

Comment:
It definitely makes sense... but i'm afraid we can't change the accessibility in a subclass =(

<img width="800" alt="screen shot 2016-03-21 at 12 02 33" src="https://cloud.githubusercontent.com/assets/1195260/13922836/ddc73650-ef5c-11e5-9e02-1e0281ff40b7.png">